src/index.html

```diff
@@ -2,11 +2,405 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <div class="logo">
+        <a class="home" href="https://www.moyo.ua/ua/">
+          <img src="/images/logo.png" alt="Moyo logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a class="nav__link" href="#">Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container_grid">
+        <div
+            class="product"
+            data-qa="card"
+        >
+          <div class="product__image-container">
+            <img
+                class="product__image"
+                src="/images/imac.jpeg"
+                alt="Apple iMac"
+            >
+          </div>
+          <h3 class="product-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="product-description">
+            Product code: 195434
+          </p>
```

Comment:
```suggestion
          <h3 class="product__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h3>

          <p class="product__description">
            Product code: 195434
          </p>
```
It can be just BEM elements